### ISSUE TITLE
Dimmable Light Issue

### ISSUE THREAD

**Issue by pattont** [damianxd/homebridge-vera#138]
I have a dimmable Z-Wave Light (Vera Plus) and everything with Homebridge and Homebride-Vera works with accessing the light and dimming it using Homekit on my phone. I have one issue though. I have created a HomeKit scene with the device so I can tell Siri to "turn off all my lights" for instance. I put this dimmable light into the scene and tell it that I want the device off when I activate the scene and upon activating the scene it does not turn it off if the device is on any percentage except 100%.

If the light is on any percentage and I click in Homekit to turn it off, it doesn't turn off even though the icon goes grey even outside the scene, I have to then click the light icon again (when it says the device is off) to turn the light on to 100%, and then click the icon once again to actually turn the light to turn off. If I am simply turning the device on or managing the dim percentage it’s fine. Just when I want to turn the light off is when I have the issue. 

Any ideas? I would have to say it's a Homebridge or Homebridge-Vera issue because I use HA Bridge to allow for Amazon Echo to talk to my devices and it manages these dimmable devices just fine.

**Comment by pattont** [comment]
I see that it’s a “known issue” any chance we can work through a debug for this? I’m a developer myself, but know little about Homebridge other than installation. 

If I can be of assistance let me know. I have numerous dimmers in my house so this bug is quite annoying!

**Comment by pattont** [comment]
Any word on this?

**Comment by spacesuitdiver** [comment]
I just tried to reproduce your issue as I thought it might be related to toggling dimmable bulbs not returning to their dimmed state but couldn't. Is this still an issue for you?

**Comment by pattont** [comment]
Yes, it’s specially with dimmable light switches. The dimmable fan switches
(which are GE Zwave Switches as well) I believe work fine. I’ll test
tomorrow and get back with you.

One thing with the testing. If I turn on the device from HomeKit to a
percentage and then turn it off it’s fine. If I turn the switch on from the
wall and then open HomeKit, it shows the percentage, but I cannot turn it
off. It shows it going off, but they light stays on. If I open HomeKit
again the switch shows the percentage again.

:-( :-( :-(



Travis
